source/server/hot_restart_nop_impl.h

```diff
@@ -5,6 +5,9 @@
 #include "envoy/server/hot_restart.h"
 
 #include "common/common/thread.h"
+#include "common/stats/stats_impl.h"
+
+#include "server/hot_restart_impl.h"
```

Comment:
Why does this include `hot_restart_impl.h`?

Reply:
it needs `blockMemHashOptions` from there.